### ISSUE TITLE
Incorrect answer on Assignment 10.1 2d

### ISSUE THREAD

**Issue by aremis9** [adyavanapalli/Stanford-Introduction-to-Mathematical-Thinking#6]
![image](https://github.com/adyavanapalli/stanford-introduction-to-mathematical-thinking/assets/72900045/24a3dd01-1726-4ce2-8a6b-b8dd1d6db0ab)

The correct answer is (3, 8] and the posted answer is (-oo,3] U (8,oo), which is incorrect.

**Comment by adyavanapalli** [comment]
Correct, I provided the complement of the given set mistakenly. Good catch. I'll fix this later tonight.
